windows/ReactTestApp/Manifest.cpp

```diff
@@ -7,6 +7,8 @@
 
 #include <nlohmann/json.hpp>
 
+using namespace nlohmann::detail;
```

Comment:
Do we really need the whole namespace? Can we reduce this to:
```suggestion
using nlohmann::json::value_t;
```